---
title: 'Archaeal Richmond Mine Acidophilic Nanoorganisms'
date: 2020-01-13T07:47:00+01:00
draft: false
---

**Archaeal Richmond Mine acidophilic nanoorganisms (ARMAN)** were first discovered in an extremely acidic mine located in northern California ([Richmond Mine at Iron Mountain](https://en.wikipedia.org/wiki/Iron_Mountain_Mine "Iron Mountain Mine")) by Brett Baker in Jill Banfield's laboratory at the [University of California Berkeley](https://en.wikipedia.org/wiki/University_of_California_Berkeley "University of California Berkeley"). These novel groups of [archaea](https://en.wikipedia.org/wiki/Archaea "Archaea") named ARMAN-1, ARMAN-2 (_[Candidatus](https://en.wikipedia.org/wiki/Candidatus "Candidatus") [Micrarchaeum acidiphilum](https://en.wikipedia.org/wiki/Micrarchaeum_acidiphilum "Micrarchaeum acidiphilum")_ ARMAN-2 ), and ARMAN-3 were missed by previous PCR-based surveys of the mine community because the ARMANs have several mismatches with commonly used [PCR](https://en.wikipedia.org/wiki/PCR "PCR") primers for [16S rRNA](https://en.wikipedia.org/wiki/16S_ribosomal_RNA "16S ribosomal RNA") genes. Baker _et al._[\[1\]](https://en.wikipedia.org/wiki/Archaeal_Richmond_Mine_acidophilic_nanoorganisms#cite_note-1) detected them in a later study using [shotgun sequencing](https://en.wikipedia.org/wiki/Shotgun_sequencing "Shotgun sequencing") of the community. The three groups were originally thought to represent three unique lineages deeply branched within the [Euryarchaeota](https://en.wikipedia.org/wiki/Euryarchaeota "Euryarchaeota"), a subgroup of the [Archaea](https://en.wikipedia.org/wiki/Archaea "Archaea"). However, this has been revised, based on more complete archaeal genomic tree, that they belong to a super phylum named DPANN.[\[2\]](https://en.wikipedia.org/wiki/Archaeal_Richmond_Mine_acidophilic_nanoorganisms#cite_note-2) The ARMAN groups now comprise deeply divergent phyla named Micrarchaeota and Parvarchaeota.[\[3\]](https://en.wikipedia.org/wiki/Archaeal_Richmond_Mine_acidophilic_nanoorganisms#cite_note-3) Their 16S rRNA genes differ by as much as 17% between the three groups. Prior to their discovery all of the [Archaea](https://en.wikipedia.org/wiki/Archaea "Archaea") shown to be associated with Iron Mountain belonged to the order [Thermoplasmatales](https://en.wikipedia.org/wiki/Thermoplasmatales "Thermoplasmatales") (e.g., _[Ferroplasma acidarmanus](https://en.wikipedia.org/wiki/Ferroplasma_acidarmanus "Ferroplasma acidarmanus")_).

Distribution
------------

Examination of different sites in the mine using fluorescent probes specific to the ARMAN groups has revealed that they are always present in communities associated with [acid mine drainage](https://en.wikipedia.org/wiki/Acid_mine_drainage "Acid mine drainage") (AMD), at Iron Mountain in northern California, that have pH < 1.5. They are usually found in low abundance (5–25%) in the community. Recently, closely related organisms have been detected in an acidic boreal mire or [bog](https://en.wikipedia.org/wiki/Bog "Bog") in Finland,[\[4\]](https://en.wikipedia.org/wiki/Archaeal_Richmond_Mine_acidophilic_nanoorganisms#cite_note-4) another acid mine drainage site in extreme environments of [Rio Tinto](https://en.wikipedia.org/wiki/Rio_Tinto_(river) "Rio Tinto (river)"), southwestern Spain[\[5\]](https://en.wikipedia.org/wiki/Archaeal_Richmond_Mine_acidophilic_nanoorganisms#cite_note-5) and from weak alkaline deep subsurface hot spring in Yunohama, Japan.[\[6\]](https://en.wikipedia.org/wiki/Archaeal_Richmond_Mine_acidophilic_nanoorganisms#cite_note-6)

Cell structure and ecology
--------------------------

Using [cryo-electron tomography](https://en.wikipedia.org/wiki/Cryo-electron_tomography "Cryo-electron tomography") an extensive 3D characterization of uncultivated ARMAN cells within mine [biofilms](https://en.wikipedia.org/wiki/Biofilms "Biofilms") has been done (Comolli _et al._ 2009[\[7\]](https://en.wikipedia.org/wiki/Archaeal_Richmond_Mine_acidophilic_nanoorganisms#cite_note-7)). This has revealed that they are right at the cell size predicted to be the lower limit for life, 0.009 µm3 and 0.04 µm3 (NRC Steering group). They also found that despite their unusually small cell size it is common to find more than one type of [virus](https://en.wikipedia.org/wiki/Virus "Virus") attached to the cells while in the biofilms. Furthermore, the cells contain on average ≈92 [ribosomes](https://en.wikipedia.org/wiki/Ribosomes "Ribosomes") per cell, whereas the average _[E. coli](https://en.wikipedia.org/wiki/E._coli "E. coli")_ cell grown in culture contains ≈10,000 ribosomes. This suggests that for ARMAN cells a much more limited number of metabolites are present in a given cell. It raises questions about what the minimal requirements are for a living cell.

3D reconstructions of ARMAN cells in the environment has revealed that a small number of them attach to other Archaea of the order [Thermoplasmatales](https://en.wikipedia.org/wiki/Thermoplasmatales "Thermoplasmatales") (Baker _et al._ 2010 [\[8\]](https://en.wikipedia.org/wiki/Archaeal_Richmond_Mine_acidophilic_nanoorganisms#cite_note-auto-8)). The Thermoplasmatales cells appear to penetrate the cell wall to the cytoplasm of the ARMAN cells.[\[9\]](https://en.wikipedia.org/wiki/Archaeal_Richmond_Mine_acidophilic_nanoorganisms#cite_note-9) The nature of this interaction hasn't been determined. It could be some sort of parasitic or symbiotic interaction. It is possible that ARMAN is getting some sort of metabolite that it is not able to produce on its own.

Genomics and proteomics
-----------------------

The genomes of three ARMAN groups have been sequenced at the DOE [Joint Genome Institute](https://en.wikipedia.org/wiki/Joint_Genome_Institute "Joint Genome Institute") during a 2006 [Community Sequencing Program (CSP)](http://www.jgi.doe.gov/sequencing/why/3423.html). These three genomes were successfully binned from the community genomic data using ESOM or Emergent [Self-Organizing Map](https://en.wikipedia.org/wiki/Self-Organizing_Map "Self-Organizing Map") clustering of tetra-nucleotide DNA signatures (Dick _et al._ 2009[\[10\]](https://en.wikipedia.org/wiki/Archaeal_Richmond_Mine_acidophilic_nanoorganisms#cite_note-10)).

The first draft of _Candidatus_ Micrarchaeum acidiphilum ARMAN-2 is ≈1 Mb (3 scaffolds, Baker _et al._). PNAS 2010.[\[8\]](https://en.wikipedia.org/wiki/Archaeal_Richmond_Mine_acidophilic_nanoorganisms#cite_note-auto-8) The ARMAN-2 has recently been closed using 454 and Solexa sequencing of other biofilms to close the gaps and is being prepared for submission to NCBI. The genomes of ARMAN-4 and ARMAN-5 (roughly 1 Mb as well) have unusually average genes sizes, similar to those seen in endosymbiotic and parasitic bacteria. This may be signature of their inter-species interactions with other Archaea in nature (Baker _et al._ 2010 [\[8\]](https://en.wikipedia.org/wiki/Archaeal_Richmond_Mine_acidophilic_nanoorganisms#cite_note-auto-8)). Furthermore, the branching of these groups near the [Euryarchaea](https://en.wikipedia.org/w/index.php?title=Euryarchaea&action=edit&redlink=1 "Euryarchaea (page does not exist)")/[Crenarchaea](https://en.wikipedia.org/wiki/Crenarchaea "Crenarchaea") divide is reflected in them sharing many genetic aspects of both Crenarchaea and Euryarchaea. Specifically they have many genes that had previously only been identified in Crenarchaea. It is difficult to elucidate many of the commonly known metabolic pathways in ARMAN due to the unusually high number of unique genes that have been identified in their genomes.

A novel type of tRNA splicing endonuclease, involved in the processing of tRNA, has been discovered in ARMAN groups 1 and 2 (Fujishima _et al._ 2011[\[11\]](https://en.wikipedia.org/wiki/Archaeal_Richmond_Mine_acidophilic_nanoorganisms#cite_note-11)). The enzyme consists of two duplicated catalytic units and one structural unit encoded on a single gene, representing a novel three-unit architecture.

References
----------

1.  **[^](https://en.wikipedia.org/wiki/Archaeal_Richmond_Mine_acidophilic_nanoorganisms#cite_ref-1)** Baker, Brett J.; et al. (2006). "Lineages of Acidophilic Archaea Revealed by Community Genomic Analysis". _[Science](https://en.wikipedia.org/wiki/Science_(journal) "Science (journal)")_. **314** (5807): 1933–1935. [Bibcode](https://en.wikipedia.org/wiki/Bibcode "Bibcode"):[2006Sci...314.1933B](https://ui.adsabs.harvard.edu/abs/2006Sci...314.1933B). [doi](https://en.wikipedia.org/wiki/Digital_object_identifier "Digital object identifier"):[10.1126/science.1132690](https://doi.org/10.1126%2Fscience.1132690). [PMID](https://en.wikipedia.org/wiki/PubMed_Identifier "PubMed Identifier") [17185602](https://pubmed.ncbi.nlm.nih.gov/17185602).
2.  **[^](https://en.wikipedia.org/wiki/Archaeal_Richmond_Mine_acidophilic_nanoorganisms#cite_ref-2)** Rinke, C; Schwientek, P; Sczyrba, A; Ivanova, NN; Anderson, IJ; Cheng, JF; Darling, A; Malfatti, S; Swan, BK; Gies, EA; Dodsworth, JA; Hedlund, BP; Tsiamis, G; Sievert, SM; Liu, WT; Eisen, JA; Hallam, SJ; Kyrpides, NC; Stepanauskas, R; Rubin, EM; Hugenholtz, P; Woyke, T (2013). "Insights into the phylogeny and coding potential of microbial dark matter". _Nature_. **499** (7459): 431–437. [Bibcode](https://en.wikipedia.org/wiki/Bibcode "Bibcode"):[2013Natur.499..431R](https://ui.adsabs.harvard.edu/abs/2013Natur.499..431R). [doi](https://en.wikipedia.org/wiki/Digital_object_identifier "Digital object identifier"):[10.1038/nature12352](https://doi.org/10.1038%2Fnature12352). [hdl](https://en.wikipedia.org/wiki/Handle_System "Handle System"):[1912/6194](https://hdl.handle.net/1912%2F6194). [PMID](https://en.wikipedia.org/wiki/PubMed_Identifier "PubMed Identifier") [23851394](https://pubmed.ncbi.nlm.nih.gov/23851394).
3.  **[^](https://en.wikipedia.org/wiki/Archaeal_Richmond_Mine_acidophilic_nanoorganisms#cite_ref-3)** Castelle, CJ; Wrighton, KC; Thomas, BC; Hug, LA; Brown, CT; Wilkins, MJ; Frischkorn, KR; Tringe, SG; Singh, A; Markillie, LM; Taylor, RC; Williams, KH; Banfield, JF (2015). "Genomic Expansion of Domain Archaea Highlights Roles for Organisms from New Phyla in Anaerobic Carbon Cycling". _Current Biology_. **25** (6): 690–701. [doi](https://en.wikipedia.org/wiki/Digital_object_identifier "Digital object identifier"):[10.1016/j.cub.2015.01.014](https://doi.org/10.1016%2Fj.cub.2015.01.014). [PMID](https://en.wikipedia.org/wiki/PubMed_Identifier "PubMed Identifier") [25702576](https://pubmed.ncbi.nlm.nih.gov/25702576).
4.  **[^](https://en.wikipedia.org/wiki/Archaeal_Richmond_Mine_acidophilic_nanoorganisms#cite_ref-4)** Juottonen _et al._ [Seasonality of rDNA- and rRNA-derived archaeal communities and methanogenic potential in a boreal mire](http://www.nature.com/ismej/journal/v2/n11/full/ismej200866a.html), ISME Journal 24 July 2008
5.  **[^](https://en.wikipedia.org/wiki/Archaeal_Richmond_Mine_acidophilic_nanoorganisms#cite_ref-5)** Amaral-Zettler _et al._ [Microbial community structure across the tree of life in the extreme Río Tinto](http://www.nature.com/ismej/journal/vaop/ncurrent/full/ismej2010101a.html), ISME Journal 2010
6.  **[^](https://en.wikipedia.org/wiki/Archaeal_Richmond_Mine_acidophilic_nanoorganisms#cite_ref-6)** Murakami _et al._ [Metatranscriptomic analysis of microbes in an ocean-front deep subsurface hot spring reveals novel small RNAs and type-specific tRNA degradation](http://aem.asm.org/content/early/2011/12/02/AEM.06811-11), Appl Environ Microbiol 2011
7.  **[^](https://en.wikipedia.org/wiki/Archaeal_Richmond_Mine_acidophilic_nanoorganisms#cite_ref-7)** Commolli, LR _et al._ [Three-dimensional analysis of the structure and ecology of a novel, ultra-small archaeon](http://www.nature.com/ismej/journal/v3/n2/full/ismej200899a.html), ISME Journal 3, 159–167.
8.  ^ [_**a**_](https://en.wikipedia.org/wiki/Archaeal_Richmond_Mine_acidophilic_nanoorganisms#cite_ref-auto_8-0) [_**b**_](https://en.wikipedia.org/wiki/Archaeal_Richmond_Mine_acidophilic_nanoorganisms#cite_ref-auto_8-1) [_**c**_](https://en.wikipedia.org/wiki/Archaeal_Richmond_Mine_acidophilic_nanoorganisms#cite_ref-auto_8-2) Baker; et al. (2010). ["Enigmatic, ultrasmall, uncultivated Archaea"](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2889320). _Proc. Natl. Acad. Sci_. **107** (19): 8806–8811. [Bibcode](https://en.wikipedia.org/wiki/Bibcode "Bibcode"):[2010PNAS..107.8806B](https://ui.adsabs.harvard.edu/abs/2010PNAS..107.8806B). [doi](https://en.wikipedia.org/wiki/Digital_object_identifier "Digital object identifier"):[10.1073/pnas.0914470107](https://doi.org/10.1073%2Fpnas.0914470107). [PMC](https://en.wikipedia.org/wiki/PubMed_Central "PubMed Central") [2889320](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2889320). [PMID](https://en.wikipedia.org/wiki/PubMed_Identifier "PubMed Identifier") [20421484](https://pubmed.ncbi.nlm.nih.gov/20421484).
9.  **[^](https://en.wikipedia.org/wiki/Archaeal_Richmond_Mine_acidophilic_nanoorganisms#cite_ref-9)** Sanders, Robert (3 May 2010). ["Weird, ultra-small microbes turn up in acidic mine drainage"](http://newscenter.berkeley.edu/2010/05/03/ARMAN_semifinal/).
10.  **[^](https://en.wikipedia.org/wiki/Archaeal_Richmond_Mine_acidophilic_nanoorganisms#cite_ref-10)** Dick _et al._ [Community-wide analysis of microbial genome sequence signatures.](http://genomebiology.com/2009/10/8/R85) Genome Biology 10:R85
11.  **[^](https://en.wikipedia.org/wiki/Archaeal_Richmond_Mine_acidophilic_nanoorganisms#cite_ref-11)** Fujishima; et al. (2011). ["A novel three-unit tRNA splicing endonuclease found in ultrasmall Archaea possesses broad substrate specificity"](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3239211). _Nucleic Acids Res_. **39** (22): 9695–704. [doi](https://en.wikipedia.org/wiki/Digital_object_identifier "Digital object identifier"):[10.1093/nar/gkr692](https://doi.org/10.1093%2Fnar%2Fgkr692). [PMC](https://en.wikipedia.org/wiki/PubMed_Central "PubMed Central") [3239211](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3239211). [PMID](https://en.wikipedia.org/wiki/PubMed_Identifier "PubMed Identifier") [21880595](https://pubmed.ncbi.nlm.nih.gov/21880595).

External links
--------------

Extant

[life](https://en.wikipedia.org/wiki/Life "Life")

phyla/divisions by domain

<img src="https://en.wikipedia.org/wiki/Special:CentralAutoLogin/start?type=1x1" alt="" title="" />

  
  
from Hacker News https://ift.tt/30gLAjR